src/gui/guiConfirmRegistration.cpp

```diff
@@ -193,10 +196,14 @@ bool GUIConfirmRegistration::processInput()
 bool GUIConfirmRegistration::OnEvent(const SEvent &event)
 {
 	if (event.EventType == EET_KEY_INPUT_EVENT) {
-		if (event.KeyInput.Key == KEY_ESCAPE && event.KeyInput.PressedDown) {
+		// clang-format off
+		if ((event.KeyInput.Key == KEY_RETURN ||
```

Comment:
Conflicts with the check in L207 below. Are you sure about removing the `KEY_ESCAPE` check here?

Reply:
Good catch, that should be `KEY_ESCAPE` not `KEY_RETURN` same in `guiPasswordChange.cpp`